conn.go

```diff
@@ -1055,6 +1055,33 @@ func (c *Conn) Create(path string, data []byte, flags int32, acl []ACL) (string,
 	return res.Path, err
 }
 
+func (c *Conn) CreateContainer(path string, data []byte, flags int32, acl []ACL) (string, error) {
+	if err := validatePath(path, flags&FlagSequence == FlagSequence); err != nil {
+		return "", err
+	}
+	if flags&FlagTTL != FlagTTL {
+		return "", ErrInvalidFlags
+	}
+
+	res := &createResponse{}
+	_, err := c.request(opCreateContainer, &CreateContainerRequest{path, data, acl, flags}, res, nil)
+	return res.Path, err
+}
+
+// ttl: ms
+func (c *Conn) CreateTTL(path string, data []byte, flags int32, acl []ACL, ttl int64) (string, error) {
```

Comment:
should our Public API in Go use time.Duration for TTL? it's a little awkward to take the int64 as input here.

Reply:
Yes, duration is better. I have revised it.